docs/yocto-project/supported-machines.md

```diff
@@ -1,7 +1,7 @@
-The **UpdateHub** support multiple **Yocto Project** releases and the latest **Yocto Project** release we offer support for is the 3.1 LTS (dunfell).
+The **UpdateHub** support multiple **Yocto Project** releases and the latest **Yocto Project** release we offer support for is the 3.2 LTS (Gatesgarth).
```

Comment:
The **UpdateHub** support multiple **Yocto Project** releases and the latest **Yocto Project** release we offer support for is the 3.2 (Gatesgarth), including the 3.1 (Dunfell) LTS release.